docker/musicbrainz-nginx/sites-available/musicbrainz.m4

```diff
@@ -0,0 +1,101 @@
+include(`config.all.m4')dnl
+resolver NGINX_RESOLVER;
```

Comment:
m4 will replace this with the definition from docker/config.all.m4. If you don't want something to be interpreted as a macro, you have to quote it like `` `NGINX_RESOLVER'``. If you need to use the start quote character in the file (a backtick by default), you need to use `changequote` to change it temporarily. The end quote character can appear without trouble as long as the start quote character doesn't precede it.